datascience/tables.py

```diff
@@ -2350,22 +2355,30 @@ def scatter(self, column_for_x, select=None, overlay=True, fit_line=False,
         options.update(vargs)
 
         x_data, y_labels =  self._split_column_and_labels(column_for_x)
-        if colors is not None:
-            y_labels.remove(self._as_label(colors))
+        if group is not None and colors is not None and group != colors:
+            warnings.warn("Do not pass both colors and group to scatter().")
+        if group is None and colors is not None:
+            # Backward compatibility
+            group = colors
+            # TODO: In a future release, warn that this is deprecated.
+            # Deprecated
+            # warnings.warn("scatter(colors=x) is deprecated. Use scatter(group=x)", FutureWarning)
```

Comment:
Let's go ahead and uncomment this. Let's change the warning a bit to say "Table#scatter(colors=x) is deprecated. Use Table#scatter(group=x)" to make it more clear what methods are being referred to.

Reply:
I'd prefer to leave this for a future release, so we can have the summer class adopt this updated version of the datascience library without triggering alarming-looking warnings.